### ISSUE TITLE
Suggestion: make it work as Win+V on windows

### ISSUE THREAD

**Issue by gersonjferreira** [Tudmotu/gnome-shell-extension-clipboard-indicator#464]
There's already a request to paste with `enter` instead of `v` (issue #412), but it would be even better to add a `Super+V` shortcut to popup the list near the text area or cursor like on Windows, and paste selected with `enter` as well.

At the moment I can mimic this by setting the `Toggle the menu` shortcut to `Super+c` (cannot use v here because it is set to show clock/agenda on gnome) and press `v` to paste. But a `Super + v' and `enter` would be much better.

Also, the paste with `v` seems quite slow for some reason.

**Comment by Tudmotu** [comment]
Yeah, this is one of those feature I've been wanting to add for a long time, but can't find any time for :sweat_smile: 
I would definitely review a PR for this feature. But it won't be easy to implement I think.

**Comment by gersonjferreira** [comment]
I only code math+physics... I have no knowledge on coding extensions and GUI interfaces.

But a "paste with enter" would be already close enough. The current "paste with v" call is not working as expected (I think), it's slow for some reason (why?) and it pastes the current selected item, instead of the highlighted item.

Some changes in these features with the popup window there a the top as it is now would be already 90% of the Win+V feature.

**Comment by banocean** [comment]
> There's already a request to paste with `enter` instead of `v` (issue #412), but it would be even better to add a `Super+V` shortcut to popup the list near the text area or cursor like on Windows, and paste selected with `enter` as well.
> 
> At the moment I can mimic this by setting the `Toggle the menu` shortcut to `Super+c` (cannot use v here because it is set to show notification list on gnome) and press `v` to paste. But a `Super + v' and `enter` would be much better.
> 
> Also, the paste with `v` seems quite slow for some reason.

you can disable the shortcut in gnome settings and then add it without any issues

**Comment by Rand0mB0y** [comment]
> you can disable the shortcut in gnome settings and then add it without any issues

@banocean How? Even after disabling the shortcut in the gnome settings, it is still not possible to choose Super + v as a shortcut in Clipboard Indicator... it's like it has been hard coded in the app to be "internally disallowed". 
I've even tried to use Input Remapper to "trick" it, but it does not work reliably, too. 
It'd be great to be able to use Super + V to show Clipboard Indicator

**Comment by CallMeAlexO** [comment]
> > you can disable the shortcut in gnome settings and then add it without any issues
> 
> [@banocean](https://github.com/banocean) How? Even after disabling the shortcut in the gnome settings, it is still not possible to choose Super + v as a shortcut in Clipboard Indicator... it's like it has been hard coded in the app to be "internally disallowed". I've even tried to use Input Remapper to "trick" it, but it does not work reliably, too. It'd be great to be able to use Super + V to show Clipboard Indicator

What works for me is modifying the shortcut value directly with dconf.

If you haven't already, install dconf-cli:
```
sudo apt install dconf-cli
```

Verify that the key exists for you:
```
dconf read /org/gnome/shell/extensions/clipboard-indicator/toggle-menu
```

And set it to Super+V:
```
dconf write /org/gnome/shell/extensions/clipboard-indicator/toggle-menu "['<Super>v']"
```

**Comment by piRGoif** [comment]
Shortcut is working for me
Also the option to display the menu right closed to the cursor is perfect !

One difference though with Microsoft Windows is in this scenario : 
- Win+V
- select an entry in the menu
- Ctrl+V 
=> in Windows you'll get the same result (old entry is pasted again with Ctrl+V)
I guess this is the issue reported in both #559 and #613 ?

**Comment by banocean** [comment]
> > you can disable the shortcut in gnome settings and then add it without any issues
> 
> [@banocean](https://github.com/banocean) How? Even after disabling the shortcut in the gnome settings, it is still not possible to choose Super + v as a shortcut in Clipboard Indicator... it's like it has been hard coded in the app to be "internally disallowed". I've even tried to use Input Remapper to "trick" it, but it does not work reliably, too. It'd be great to be able to use Super + V to show Clipboard Indicator

You sure? I just set it right now and it worked, not sure where is it supposed to be hard coded
